content/applications/marketing/surveys/live_session.rst

```diff
@@ -0,0 +1,218 @@
+====================
+Live Session surveys
+====================
+
+With the Odoo *Surveys* application, users can enhance in-person demonstrations and presentations
+with the *Live Session* feature.
+
+*Live Session* surveys function the same way as a normal survey, but with a host (or moderator), who
+presents the questions to participants, reveals their responses in real-time, and controls the tempo
+of the overall survey.
+
+In *Live Session* surveys, participants access the survey experience via a custom URL, and sign-in
```

Comment:
```suggestion
In *Live Session* surveys, participants access the survey experience via a custom URL, and sign in
```
Think this is correct here - they sign in, instead of adding their sign-in info